generator/types.go

```diff
@@ -429,6 +429,12 @@ type FieldKind int
 
 const (
 	// Basic is a field with a basic type.
+	// On top of the Go basic types, we consider Basic as well the following
+	// types:
+	//  - time.Time
+	//  - time.Duration
+	//  - kallax.ID
```

Comment:
are you sure about `kallax.ID`?

I think it matches this processor rule:
```
func (p *Processor) processField(field *Field, typ types.Type, ...) {
    switch typ := typ.(type) {
    ...
    case *types.Named:
        ...
        if p.isSQLType(types.NewPointer(typ)) {
            field.Kind = Interface
            return
        }
        ...
    ...
    }
}
```
so it would be an `Interface FieldKind`, isn't it?